debian/patches/avoiding-building-and-win-when-testing.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/bin/test-unit b/bin/test-unit
```

Comment:
FWIW, I helped @mirespace a bit with this package and I think this patch is not necessary.  The testsuite won't be invoked during build time for now, and even when/if they are ever enabled, I think the idea will be to invoke them manually on `override_dh_auto_test` instead of relying on these scripts provided by upstream.